src/js/containers/covid19/recipient/map/MapContainer.jsx

```diff
@@ -390,6 +392,10 @@ export class MapContainer extends React.Component {
                 className="results-visualization-geo-section"
                 id="results-section-geo"
                 aria-label="Spending by Geography">
+                <AwardFilterButtons
+                    onClick={this.updateawardTypeFilter}
```

Comment:
```suggestion
                    onClick={this.updateAwardTypeFilter}
```
There are some other functions in this file too that could be properly camel-cased to improve readability